tests/saitests/py3/sai_base_test.py

```diff
@@ -177,8 +178,6 @@ def sai_thrift_port_tx_enable(self, client, asic_type, port_list, target='dst',
             assert 'Success rv = 0' in stdOut[1], "enable wd failed '{}' on asic '{}' on '{}'".format(
                     cmd, self.src_asic_index, self.src_server_ip)
 
-        sai_thrift_port_tx_enable(client, asic_type, port_list, target=target)
-
     def sai_thrift_port_tx_disable(self, client, asic_type, port_list, target='dst'):
         if self.platform_asic and self.platform_asic == "broadcom-dnx":
             # need to enable watchdog on the source asic using cint script
```

Comment:
Please check if there is any restriction on when credit-watchdog can be enabled/disabled in terms of traffic in the system.